service/src/main/java/com/fecklessweasel/service/objectmodel/FileMetadata.java

```diff
@@ -0,0 +1,281 @@
+   *@return The FileMetadata object representation of the row found
+   */
+  public static FileMetadata lookup(Connection sql, int fid) throws ServiceException{
+    OMUtil.sqlCheck(sql);
+    ResultSet result = FileMetadataTable.lookUpFile(sql, fid);
+
+    try{
+        //if no tuples returned, throw error.
+        if (!result.next()) { throw new ServiceException(ServiceStatus.APP_FILE_NOT_EXIST); }
+
+        FileMetadata fileData = new FileMetadata(result.getInt("fid"),
+                                                 result.getString("user"),
+                                                 result.getString("course"),
+                                                 result.getString("university"),
+                                                 result.getDate("creation_date"),
+                                                 result.getInt("rating"));
+        result.close();
+        return fileData;
+    } catch (SQLException ex) {
+      throw new ServiceException(ServiceStatus.DATABASE_ERROR, ex);
+    }
+  }
+
+  /**
+   * Look up all files in a specified course
+   *@param sql The Sql connection to the FecklessWeaselDB
+   *@param course The course whose file info we want
+   *@param university The University that has the course
+   *@return A List of FileMetadata objects that represent each files info
+   */
+  public static List<FileMetadata> lookUpCourseFiles(Connection sql, String course, String university)
+    throws ServiceException{
+    OMUtil.sqlCheck(sql);
+    OMUtil.nullCheck(course);
+    OMUtil.nullCheck(university);
+
+    ResultSet results = FileMetadataTable.lookUpCourseFiles(sql, course, university);
+    ArrayList<FileMetadata> listOfFiles = new ArrayList<FileMetadata>();
+
+    try{
+      while( results.next() ){
+        FileMetadata fileData = new FileMetadata(results.getInt("fid"),
+                                                 results.getString("user"),
+                                                 results.getString("course"),
+                                                 results.getString("university"),
+                                                 results.getDate("creation_date"),
+                                                 results.getInt("rating"));
+        listOfFiles.add(fileData);
+      }
+      results.close();
+      return listOfFiles;
+    } catch (SQLException ex){
+      throw new ServiceException(ServiceStatus.DATABASE_ERROR, ex);
+    }
+  }
+  /**
+   * Look up all files belonging to a specific user
+   *@param sql The Sql connection to the FecklessWeaselDB
+   *@param user The User whos file info we are interested in
+   *@return A List of FileMetadata objects that represent each files info
+   */
+  public static List<FileMetadata> lookUpUserFiles(Connection sql, String user)
+    throws ServiceException{
+    OMUtil.sqlCheck(sql);
+    OMUtil.nullCheck(user);
+
+    ResultSet results = FileMetadataTable.lookUpUsersFiles(sql, user);
```

Comment:
You have both FileMetadataTable.lookupUsersFiles() and FileMetaData.lookupuUserFiles(). You should probably decide on one name or the other and update them accordingly.